scripts/CMakeLists.txt

```diff
@@ -185,6 +185,12 @@ ELSE()
   SET(localstatedir ${MYSQL_DATADIR})
 ENDIF()
 
+IF(DEB OR RPM)
+  SET(skip_install_test_db "1")
+ELSE()
+  SET(skip_install_test_db "NULL")
```

Comment:
Why NULL and not 0?

Reply:
because NULL IS NULL needed to be true in the sql. 0 IS NULL is false (or is it late and I'm getting even this wrong)..

Reply:
`0 IS NOT NULL`, I just thought we'll have more consistent code here. But if replacing this with 0 requires more code, then we can live with it.

Reply:
Allowed for 0 rather than null in next commit however allowed for NULL just in case the sql is used without setting the session variable for compatibility against anything outside the repository. Hopefully not likely but easy enough to account for.